xlators/cluster/afr/src/afr-self-heal-metadata.c

```diff
@@ -85,6 +88,9 @@ __afr_selfheal_metadata_do(call_frame_t *frame, xlator_t *this, inode_t *inode,
                 healed_sinks[i] = 0;
         }
 
+        ret = afr_set_heal_outcast(xattr, outcast, AFR_METADATA_TRANSACTION, 0);
```

Comment:
I didn't see any outcast marking on bad bricks at the start of heal. Is that a deliberate choice? If yes why?

Reply:
@rafikc30 It can so happen that only the metadata is modified while the brick is down. When metadata heal is triggered on that file, there won't be outcast marking. Is that intended?

Reply:
@pranithk Since the outcast is set on the file where the healing is required, When the brick is down we won't be able to set the outcast (of course we can do this for halo replication, may be as an async fop). I believe it requires more brainstorming, so I thought may be that can be implemented along with the full outcast implementation, Where we can also consider the data part.